src/h5web/visualizations/MappedHeatmapVis.tsx

```diff
@@ -24,13 +24,22 @@ function MappedHeatmapVis(props: Props): ReactElement {
     keepAspectRatio,
     showGrid,
     resetDomains,
+    autoScale,
+    disableAutoScale,
   } = useHeatmapConfig();
 
   const baseArray = useBaseArray(dataset, value);
   const dataArray = useMappedArray(baseArray, mapperState);
-  const dataDomain = useDataDomain(dataArray.data as number[]);
+  const dataDomain = useDataDomain(
+    (autoScale ? dataArray.data : baseArray.data) as number[]
+  );
   const prevDataDomain = usePrevious(dataDomain);
 
+  // Disable `autoScale` for 2D datasets (baseArray and dataArray are the same)
```

Comment:
I wouldn't say the two arrays are the same, since they may be remapped (D0 <-> D1) -- so I would remove the part in parens.

Reply:
:fishing_pole_and_fish: